acl_test.go

```diff
@@ -75,7 +75,8 @@ func TestACLs(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if len(lsws) != 1 {
+
+	if len(lsws) == 0 {
```

Comment:
i don't think that this is good solution. for example - you run second test and lsw not created, but you have older lsw....
best to do - count number of lsw before creating and after.

Reply:
count logic itself needs refactoring.  Right now we are using multiple LSW and hence hardcoding to ==1 doesn't make sense. Hence, I kept 0. To avoid check by length, we should get list of switches and then iterate by ls.Name==lsw to do actual confirmation if the switch is deleted/created. Similar is needed for all entities. Hence, I didnt handle it in this patch. Let me know and I can amend in same patch.

Reply:
@vtolstov : Done. PTAL.